wordpress.org/public_html/wp-content/plugins/trac-notifications/trac-components.php

```diff
@@ -693,7 +722,7 @@ function component_table_row( $post ) {
 		static $once = true;
 		if ( $once ) {
 			$once = false;
-			echo '<thead><tr><td>Component</td><td>Tickets</td><td>7 Days</td><td>0&nbsp;Replies</td><td>Maintainers</td></tr></thead>';
+			echo '<thead><tr><td>Component</td><td>Tickets</td><td>7 Days</td><td>0&nbsp;Replies</td><td>Slack</td><td>Maintainers</td></tr></thead>';
```

Comment:
Adding this extra column unfortunately doesn't really work due to the width of the column:

<img width="820" alt="Screenshot 2023-03-16 at 12 48 55 pm" src="https://user-images.githubusercontent.com/767313/225497662-42640296-0a1c-4d56-9051-74b95e29009f.png">

Reply:
The alternative screenshot above was done with this code, the inline `float: right` should probably be a `class="slack-float-right"` or something i guess.
```diff
 		}

 		echo '<tr>';
+		echo '<td>';
 		if ( $post->post_parent ) {
-			echo '<td>&mdash; <a href="' . get_permalink() . '">' . $post->post_title . '</a></td>';
+			echo '&mdash; <a href="' . get_permalink() . '">' . $post->post_title . '</a>';
 		} else {
-			echo '<td><a href="' . get_permalink() . '"><strong>' . $post->post_title . '</strong></a></td>';
+			echo '<a href="' . get_permalink() . '"><strong>' . $post->post_title . '</strong></a>';
 		}

+		$slack_channel = get_post_meta( $post->ID, '_slack_channel', true );
+		if ( ! empty( $slack_channel ) ) {
+			echo '<span style="float: right"><a href="' . $this->slack_channel_url( $slack_channel ) . '"><span class="screen-reader-text">Join the ' . $component . ' Slack channel</span></a></span>';
+		}
+
+		echo '</td>';
+
 		$open_tickets = 0;
 		if ( ! empty( $this->breakdown_component_type[ $component ] ) ) {
 			$open_tickets = array_sum( $this->breakdown_component_type[ $component ] );
```